eng/scripts/Language-Settings.ps1

```diff
@@ -0,0 +1,56 @@
+$Language = "c"
+$Lang = "c"
+$PackageRepository = "C"
+$packagePattern = "*.json"
+$MetadataUri = ""
+
+function Extract-c-PkgProperties ($pkgPath, $serviceName, $pkgName)
+{
+  return null
+}
+
+# Parse out package publishing information given a nupkg ZIP format.
+function Parse-c-Package($pkg, $workingDirectory) 
+{
+  $packageInfo = Get-Content -Raw -Path $pkg | ConvertFrom-JSON
+  $packageArtifactLocation = (Get-ItemProperty $pkg).Directory.FullName
+  $releaseNotes = ""
+  $readmeContent = ""
+
+  $pkgVersion = $packageInfo.version
+
+  $changeLogLoc = @(Get-ChildItem -Path $packageArtifactLocation -Recurse -Include "CHANGELOG.md")[0]
+  if ($changeLogLoc)
+  {
+    $releaseNotes = Get-ChangeLogEntryAsString -ChangeLogLocation $changeLogLoc -VersionString $pkgVersion
+  }
+  
+  $readmeContentLoc = @(Get-ChildItem -Path $packageArtifactLocation -Recurse -Include "README.md")[0]
+  if ($readmeContentLoc) 
+  {
+    $readmeContent = Get-Content -Raw $readmeContentLoc
+  }
+
+  return New-Object PSObject -Property @{
+    PackageId      = 'azure-sdk-for-c'
+    PackageVersion = $pkgVersion
+    # Artifact info is always considered deployable for C becasue it is not
+    # deployed anywhere. Dealing with duplicate tags happens downstream in
+    # CheckArtifactShaAgainstTagsList
+    Deployable     = $true
+    ReleaseNotes   = $releaseNotes
+  }
+}
+
+# Stage and Upload Docs to blob Storage
+function StageAndUpload-c-Docs ()
+{
+    # The documentation publishing process for C differs from the other
+    # langauges in this file because this script is invoked for the whole SDK
+    # publishing. It is not, for example, invoked once per service publishing.
+    # There is a similar situation for other langauge publishing steps above...
+    # Those loops are left over from previous versions of this script which were
+    # used to publish multiple docs packages in a single invocation.
+    $pkgInfo = Get-Content $DocLocation/package-info.json | ConvertFrom-Json
+    Upload-Blobs -DocDir $DocLocation -PkgName 'docs' -DocVersion $pkgInfo.version
+}
```

Comment:
nit: add a new line

https://0.readthedocs.io/en/latest/git.html#check-there-is-a-new-line-at-end-of-file